meta-luneos/recipes-webos-ose/sam/sam/0008-NativeContainer-configure-native-apps.patch

```diff
@@ -22,7 +22,6 @@ index b01a5ef..793f030 100644
 +        if(runningApp->getLaunchPoint()->getAppDesc()->useLuneOSStyle())
 +            runningApp->getLinuxProcess().addEnv("QT_QUICK_CONTROLS_STYLE", "QtQuick.Controls.LuneOS");
 +        runningApp->getLinuxProcess().addEnv("QT_QPA_PLATFORM", "wayland-egl");
```

Comment:
same here, TBC

Reply:
theoretically, it should be webos-wayland, so the app launched can handle webos window attributes and such, but if they are native, they probably don't interface to the EOS library and probably don't care.  but they could...